app/models/manageiq/providers/openshift/infra_manager/provision_workflow.rb

```diff
@@ -4,6 +4,6 @@ def self.provider_model
   end
 
   def dialog_name_from_automate(message = 'get_dialog_name')
-    super(message, {'platform' => 'openshift'})
+    super(message, "openshift")
```

Comment:
`super` call here did not match the parameters of method in the kubevirt provider https://github.com/ManageIQ/manageiq-providers-kubevirt/pull/250/files#diff-ed8161fa01fc7bedf7e66a09c7a3b212b9c4452967bb912425c9e4ee1957a7ebR32